content/blog/keda-2.0-alpha.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "Give KEDA 2.0 (Alpha) a test drive"
+date = 2020-08-19 # TODO: Update with actual date
+author = "Tom Kerkhove"
++++
+
+Today, we are happy to share that our first **alpha version of KEDA 2.0 is available**! 🎊
+
+# Highlights
+
+With this release, we are shipping majority of our planned features.
+
+Here are some highlights:
+
+- **Making scaling more powerful**
+    - Introduction of `ScaledJob` ([docs](https://keda.sh/docs/2.0/concepts/scaling-jobs/))
+    - Support for scaling Deployments, Stateful Sets and/or any Custom Resources ([docs](https://keda.sh/docs/2.0/concepts/scaling-deployments/))
+    - Support for scaling on standard resource metrics (CPU/Memory)
+    - Support for multiple triggers in a single `ScaledObject` ([docs](https://keda.sh/docs/2.0/concepts/scaling-deployments/))
+    - Support for scaling to original replica count after deleting `ScaledObject` ([docs](https://keda.sh/docs/2.0/concepts/scaling-deployments/))
+    - Support for controling scaling behavior of underlying HPA
+- **Easier to operate KEDA**
+    - Introduction of readiness and liveness probes
+    - Introduction of Prometheus metrics for Metrics Server ([docs](https://keda.sh/docs/2.0/operate/))
+    - Provide more information when quering KEDA resources with `kubectl`
+- **Extensibility**
+    - Introduction of External Push scaler ([docs](https://keda.sh/docs/2.0/scalers/external-push/))
+    - Provide Go client
+
+For a full list of changes, we highly recommend going through [our changelog](https://github.com/kedacore/keda/blob/v2/CHANGELOG.md#v200)!
+
+# Getting started
+
+TODO but need input, just list Helm? This legit?
```

Comment:
Happy to have input here on what's expected, just added Helm (but didn't test it as it's not shipped yet)

Reply:
Something like this could work: `kubectl apply -f https://github.com/kedacore/keda/releases/download/v2.0.0/keda-2.0.0.yaml`

The version should be changed to what we are going to ship eg. something like `2.0.0.alpha` or `2.0.0.beta` ..

Reply:
Used `https://github.com/kedacore/keda/releases/download/v2.0.0-beta/keda-2.0.0-beta.yaml` for now but can change if you want.

I presume this will work because https://github.com/kedacore/keda/releases/download/v1.5.0/keda-1.5.0.yaml doesn't as we only ship a ZIP.

https://github.com/kedacore/keda/blob/v1.5.0/Makefile#L79